CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Ubirch Testkit - Changelog
+
+## [1.2.0] - 2021-03-31
+### Added
+- This changelog and a patch level in versioning.
+- Handling of unsolicited messages from modem as needed with the newer pycom firmware.
+
+### Changed
+- Update pycom firmware to version v20.2.r4.
```

Comment:
Here is a mistake in the version. should be v1.20.2.r4
Also we changed the modem firmware, didn't we? This does not show up

Reply:
Ok, fixed the version and added a note on the modem firmware version.